data/coalition/heliarch intro.txt

```diff
@@ -1849,7 +1849,7 @@ mission "Heliarch Drills 2-C"
 		not "assisting lunarium"
 	on offer
 		conversation
-			`Once again, you managed to avoid the Hunters while landing on <origin>, and a local Heliarch agent informs Magister Lotuora of your success. "Excellent work, Captain <last>," she says when she contacts you. "Now, talked with some contacts I have, and one more drill for you to help with, I have arranged. Return to <planet> at your earliest convenience, you may." She signs off, but calls back seconds after. "Ah, forgive me Captain. Forgotten something, I had. Heard of the recent drills, some colleagues did, and readied their own Hunters to participate, they now have. Avoid them on your return to <planet>, as per usual, you should."`
+			`Once again, you managed to avoid the Hunters while landing on <origin>, and a local Heliarch agent informs Magister Lotuora of your success. "Excellent work, Captain <last>," she says when she contacts you. "Now, talked with some contacts I have, and one more drill for you to help with, I have arranged. Return to <planet> at your earliest convenience, you may." She signs off, but calls back seconds after. "Ah, forgive me you must, Captain. Forgotten something, I had. Hear of the recent drills, some colleagues did, and readied their own Hunters to participate, they now have. Avoid them on your return to <planet>, as per usual, you should."`
```

Comment:
```suggestion
			`Once again, you managed to avoid the Hunters while landing on <origin>, and a local Heliarch agent informs Magister Lotuora of your success. "Excellent work, Captain <last>," she says when she contacts you. "Now, talked with some contacts I have, and one more drill for you to help with, I have arranged. Return to <planet> at your earliest convenience, you may." She signs off but calls back seconds after. "Ah, forgive me you must, Captain. Forgotten something, I had. Hear of the recent drills, some colleagues did, and readied their own Hunters to participate, they now have. Avoid them on your return to <planet>, as per usual, you should."`
```

Reply:
Piggybacking on my previous comment about independent clauses – you can keep the comma if you want, but then you have to add a subject on the other side of the comma.

`She signs off, but then she calls back again seconds later` for example.

Reply:
I understand what you're going for. That mental narration is a common source of comma splices; I have to self-correct myself from doing it all the time.

If you want the same effect, you can use an ellipsis (`...`), which would be grammatically correct and still evoke a pause.

```suggestion
			`Once again, you managed to avoid the Hunters while landing on <origin>, and a local Heliarch agent informs Magister Lotuora of your success. "Excellent work, Captain <last>," she says when she contacts you. "Now, talked with some contacts I have, and one more drill for you to help with, I have arranged. Return to <planet> at your earliest convenience, you may." She signs off... and calls back seconds after. "Ah, forgive me you must, Captain. Forgotten something, I had. Hear of the recent drills, some colleagues did, and readied their own Hunters to participate, they now have. Avoid them on your return to <planet>, as per usual, you should."`
```